pgxn/neon/neon--1.4--1.5.sql

```diff
@@ -0,0 +1,35 @@
+\echo Use "ALTER EXTENSION neon UPDATE TO '1.5'" to load this file. \quit
+
+
+CREATE FUNCTION get_backend_perf_counters()
+RETURNS SETOF RECORD
+AS 'MODULE_PATHNAME', 'neon_get_backend_perf_counters'
+LANGUAGE C PARALLEL SAFE;
+
+CREATE FUNCTION get_perf_counters()
+RETURNS SETOF RECORD
+AS 'MODULE_PATHNAME', 'neon_get_perf_counters'
+LANGUAGE C PARALLEL SAFE;
+
+-- Show various metrics, for each backend. Note that the values are not
+-- reset when a backend exits. When a new backend starts with the backend
+-- ID, it will continue accumulating the values from where the old backend
```

Comment:
NIT: The current implementation seems ambiguous to me. I see that you reused the same shared struct for both cumulative and per-backend stats; that's why we cannot reset the counters, but is the end result handy? Procno is some internal concept that is hard to reason about

Reply:
The view exposes 'pid' as well, which is pretty convenient to use, e.g. to show stats of just your own backend
```
select * from neon_backend_perf_counters where pid=pg_backend_pid();
```